dyld[48091]: Library not loaded: @executable_path/../lib/libpython3.12.dylib
  Referenced from: <4C4C44E3-5555-3144-A152-F415406B5622> /Users/a1/Library/Application Support/pypoetry/venv/bin/python3.12
  Reason: tried: '/Users/a1/Library/Application Support/pypoetry/venv/lib/libpython3.12.dylib' (no such file)

Traceback:

  File "<stdin>", line 937, in main
  File "<stdin>", line 576, in run
